etc/php/tools/reinstall_storefront

```diff
@@ -22,23 +22,19 @@ fi
 
 for edition in ${INSTALLED_REPOS}; do
 
-   if [[ ${edition} == "magento2ce" ]] || [[ ${edition} == "graphql" ]] || [[ ${edition} == "storefront-message-broker" ]] || [[ ${edition} == "storefront-search-ce" ]] || [[ ${edition} == "storefront-pricing-ce" ]]; then
+   if [[ ${edition} == "magento2ce" ]] || [[ ${edition} == "storefront-libraries" ]] || [[ ${edition} == "graphql" ]] || [[ ${edition} == "storefront-message-broker" ]] || [[ ${edition} == "storefront-search-ce" ]] || [[ ${edition} == "storefront-pricing-ce" ]]; then
       continue
    fi
 
    # copy into app/code/Magento
    if [[ ${edition} == "inventory" ]]; then
       dst="/var/www/magento2ce/app/code/Magento"
-   else
-      dst="/var/www/magento2ce"
```

Comment:
if some repo will be used after inventory/grpc they will use incorrect destination, dst will be the same next time. It's better move `dst="/var/www/magento2ce"` at the start of the cycle